modules/exploits/linux/http/netgear_r7000_cgibin_exec.rb

```diff
@@ -40,7 +40,8 @@ def initialize(info = {})
           'Space'       => 1024,
           'DisableNops' => true,
           'BadChars'    => "\x20"
-        }
+        },
+      'DefaultOptions'  => { 'WfsDelay' => 10}
```

Comment:
You can, but you'd have to use `CmdStager`, and that's more code. I doubt `ARCH_CMD` would work reliably here due to all the badchars, lack of space, and stripped-down BusyBox (might be able to do something with `telnet`).

I can whip up a PR for you. Will have to wait until later today. Thanks for the fast response!